includes/class-create-block-theme-api.php

```diff
@@ -481,6 +492,33 @@ function rest_save_theme( $request ) {
 		);
 	}
 
+	/**
+	 * Get a list of all the font families used in the theme.
+	 *
+	 * It includes the font families from the theme.json data (theme.json file + global styles) and the theme style variations.
+	 * The font families with font faces containing src urls relative to the theme folder are converted to absolute urls.
+	 */
+	function rest_get_font_families( $request ) {
+		$font_families = CBT_Theme_Fonts::get_all_fonts();
+
+		// Iterates through the font families and makes the urls absolute to use in the frontend code.
+		foreach ( $font_families as &$font_family ) {
+			if ( isset( $font_family['fontFace'] ) ) {
+				foreach ( $font_family['fontFace'] as &$font_face ) {
+					$font_face['src'] = CBT_Theme_Fonts::make_theme_font_src_absolute( $font_face['src'] );
+				}
+			}
+		}
```

Comment:
I think that in general we have been keeping the REST route handlers free of data processing logic by calling the functions that perform actions on data instead of processing data directly in the handler; it might be worth it to move this `foreach` into its own function too, or perhaps refactor `get_all_fonts` so that it will call `make_theme_font_src_absolute` itself.

Reply:
:+1: Ok, I moved that logic into get_all_fonts. It also make possible to make `make_theme_font_src_absolute` access private. https://github.com/WordPress/create-block-theme/pull/648/commits/55cf62ea5644095da22d73b02dcf8292b9e8dbaf